imsc1/ttml/timing/TimeExpressions001.ttml

```diff
@@ -7,17 +7,17 @@
 	</head>
 	<body>
 		<div timeContainer="seq">			
-			<p dur="1.2s">1.2s = 1.2s</p>
-			<p dur="1.2m">1.2m = 72s</p>
-			<p dur="1.2h">1.2h = 4320s</p>
-			<p dur="24f">24f = 1.001s</p>
-			<p dur="120t">120t = 2s</p>
-			<p dur="01:02:03">01:02:03 = 3723s</p>
-			<p dur="01:02:03.235">01:02:03.235 = 3723.235s</p>
-			<p dur="01:02:03.2350">01:02:03.2350 = 3723.235s</p>
-			<p dur="01:02:03:20">01:02:03:20 = 3723.83416667s</p>
-			<p dur="100:00:00.1">100:00:00.1 = 360000.1s</p>
-			<p dur="100:00:00:100">100:00:00:100 = 360000s</p>
+			<p begin="0s" end="1.2s">1.2s = 1.2s</p>
```

Comment:
From where does the constraint that begin and end attributes must be specified originate? I can't see it in IMSC or TTML. Aside from the frame component being out of bounds in line 20 I can't see why the original test was wrong.

Reply:
>For any content element that contains br elements or text nodes or a smpte:backgroundImage attribute, both the begin attribute and one of either the end or dur attributes should be specified on the content element or at least one of its ancestors.

Under `#timing` at https://www.w3.org/TR/ttml-imsc1.0.1/

Reply:
Right, thanks for the pointer, I'd forgotten about that.